flyway/scxa/migrations/V9__scxa-allow-generic-markers.sql

```diff
@@ -0,0 +1,40 @@
+
+CREATE TABLE scxa_cell_group
+(
+  id SERIAL not null PRIMARY KEY,
+  experiment_accession VARCHAR(255) NOT NULL,
+  variable VARCHAR(255) NOT NULL,
+  value VARCHAR(255) NOT NULL,
+  UNIQUE (experiment_accession, variable, value)  
+);
+
+CREATE TABLE scxa_cell_group_membership
+(
+  experiment_accession VARCHAR(255) NOT NULL,
+  cell_id VARCHAR(255) NOT NULL,
+  cell_group_id integer references scxa_cell_group(id),
+  CONSTRAINT scxa_cell_groups_experiment_accession_cell_id_cell_group_id_pk
+    PRIMARY KEY (experiment_accession, cell_id, cell_group_id)
+);
+
+CREATE TABLE scxa_cell_group_marker_genes
+(
+  id SERIAL not null PRIMARY KEY,
+  gene_id VARCHAR(255) NOT NULL,
+  cell_group_id integer references scxa_cell_group(id),
+  marker_probability DOUBLE PRECISION,
+  UNIQUE (gene_id, cell_group_id)  
+);
+
+-- Create a table from data prepared during bundling, replacing the previously used materialised view
+
+CREATE TABLE scxa_cell_group_marker_gene_stats
+(
+  gene_id VARCHAR(255) NOT NULL,
```

Comment:
What about the expression of the genes in the other groups where they are not marker genes? Remember that in the heatmap, for gene A that is a marker gene in group Ga, you will also need summarised expression in groups Gb, Gc, ..., Gn. Conversely, for gene B, which is a marker in Gb, you will also need summarised expression in Ga... Gn.

Reply:
ahhh, ok, I see how this is covered.